src/oidcop/endpoint.py

```diff
@@ -206,14 +206,6 @@ def parse_request(
             request=req, client_id=_client_id, http_info=http_info, **kwargs
         )
 
-    def get_client_id_from_token(
```

Comment:
this is a callable and not a property, at line 94 you create instead an attribute

are we sure that's won't be faulty?

Reply:
I changed it so that it's only defined in the classes that need it